### PR TITLE
admin action tests

### LINKED ISSUE #505: Admin Filtering E2E Test
- Make tests for the filtering
- When a filter is deselected then the updated objects should replicate accordingly

### LINKED ISSUE #516: Admin Selection E2E Test
- When a user selects all the entries, or just some of the entires, make sure the correct entries are selected

### LINKED ISSUE #522: Admin Searching E2E Test
- When searching for a user, the appropriate results should show up

### LINKED ISSUE #528: Admin Remove E2E Test
- When a user removes certain entries, then make sure the results are removed from the dataset

### LINKED ISSUE #534: Admin Actions E2E Test
- When applying different action buttons, the dataset should reflect accordingly

### PR DESCRIPTION
**Pull request by menthy-wu**
image
image
image
image

### CODE REVIEW

File: src/app/layout.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ export default function RootLayout({ children, session }) {
         <body className={`${poppins.variable} flex flex-col lg:flex-row`}>
           <Toaster />
           <Navigation />
-          <div className="flex justify-center items-start w-full bg-hackathon-page z-0 h-screen">
+          <div className="flex justify-center items-start w-full bg-hackathon-page z-0 h-screen pt-12 lg:pt-0">
```
Comment (shahdivyank) [inline src/app/layout.js:24]: what is this for?

File: src/components/Admin/Button.jsx [hunk 2]
```diff
@@ -3,6 +3,7 @@ import { colors } from "@/data/admin/buttonColor";
 const Button = ({ color, text, onClick, size }) => {
   return (
     <button
+      data-cy={`${text}Button`}
```
Comment (shahdivyank) [inline src/components/Admin/Button.jsx]: can we do snake case

File: src/components/Admin/Filters.jsx [hunk 3]
```diff
@@ -28,7 +28,12 @@ const Filters = ({ filters, setFilters, setObjects, objects, input }) => {
   return (
     <Row className="w-fit">
       {Object.entries(filters).map(([filter, value], index) => (
-        <Col className="px-1" key={index} onClick={() => handleClick(filter)}>
+        <Col
+          className="px-1"
+          key={index}
+          onClick={() => handleClick(filter)}
+          data-cy={filter + (filter.endsWith("t") ? "ed" : "") + "Filter"}
```
Comment (shahdivyank) [inline src/components/Admin/Filters.jsx]: snake case

File: src/components/Admin/Toolbar.jsx [hunk 4]
```diff
@@ -130,9 +130,9 @@ const Toolbar = ({
     .replace(/\s+/g, "_");
 
   return (
-    <div className="w-full flex items-center">
+    <div className="w-full flex items-center" data-cy="toolbar">
       <div className="w-2/3 flex items-center">
-        <div className="mr-4">
+        <div className="mr-4" data-cy="selectAll">
```
Comment (shahdivyank) [inline src/components/Admin/Toolbar.jsx]: snake case

**Comment by shahdivyank** [comment]
you can do snake case with underscores or hyphens

File: cypress/e2e/admin/admin/AdminActions.cy.js [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+        .find('[data-cy="checkbox"]')
+        .click();
+    cy.get('[data-cy="toolbar"]').find('[data-cy="reject_tag"]').click();
+    for (let i = 0; i < 5; i++)
+      cy.get(`[data-cy="${adminList[i].uid}"]`)
+        .find('[data-cy="rejected_tag"]')
+        .should("exist");
+  });
+
+  it("Reset Action", () => {
```
Comment (shahdivyank) [inline cypress/e2e/admin/admin/AdminActions.cy.js]: remove this one, cause we lw might remove reset button altogether

File: cypress/e2e/admin/admin/AdminActions.cy.js [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+    cy.get('[data-cy="toolbar"]').find("input").type(adminList[0].name);
+    cy.get('[data-cy="toolbar"]').find("form").submit();
+    cy.get('[data-cy="toolbar"]').find('[data-cy="reset_tag"]').click();
+    adminList.forEach((admin) =>
+      cy.get(`[data-cy="${admin.uid}"]`).should("exist")
+    );
+  });
```
Comment (shahdivyank) [inline cypress/e2e/admin/admin/AdminActions.cy.js]:
```suggestion
  });
  
```

File: cypress/e2e/admin/admin/AdminFilter.cy.js [hunk 7]
```diff
@@ -0,0 +1,110 @@
+import { adminList } from "../../../../src/data/mock/admin";
+describe("Admin Filter", () => {
```
Comment (shahdivyank) [inline cypress/e2e/admin/admin/AdminFilter.cy.js]:
```suggestion

describe("Admin Filter", () => {
```

File: cypress/e2e/admin/admin/AdminFilter.cy.js [hunk 8]
```diff
@@ -0,0 +1,111 @@
+import { adminList } from "../../../../src/data/mock/admin";
+
+describe("Admin Filter", () => {
+  beforeEach(() => {
+    cy.login("admin");
+    cy.visit("/");
+    cy.wait("@session");
+    cy.visit("/admin/admin");
+  });
+  it("Filter Default Color", () => {
```
Comment (shahdivyank) [inline cypress/e2e/admin/admin/AdminFilter.cy.js]:
```suggestion

  it("Filter Default Color", () => {
```

File: cypress/e2e/admin/admin/AdminSearching.cy.js [hunk 9]
```diff
@@ -0,0 +1,20 @@
+import { adminList } from "../../../../src/data/mock/admin";
+describe("Admin Searching", () => {
```
Comment (shahdivyank) [inline cypress/e2e/admin/admin/AdminSearching.cy.js]:
```suggestion

describe("Admin Searching", () => {
```

File: cypress/e2e/admin/admin/AdminSearching.cy.js [hunk 10]
```diff
@@ -0,0 +1,20 @@
...
+  beforeEach(() => {
+    cy.login("admin");
+    cy.visit("/");
+    cy.wait("@session");
+    cy.visit("/admin/admin");
+  });
+  it("Input Not Exist Action", () => {
```
Comment (shahdivyank) [inline cypress/e2e/admin/admin/AdminSearching.cy.js]:
```suggestion

    it("Input Not Exist Action", () => {
```

File: cypress/e2e/admin/admin/AdminSelecting.cy.js [hunk 11]
```diff
@@ -0,0 +1,27 @@
+import { adminList } from "../../../../src/data/mock/admin";
+describe("Admin Selecting", () => {
```
Comment (shahdivyank) [inline cypress/e2e/admin/admin/AdminSelecting.cy.js]:
```suggestion

describe("Admin Selecting", () => {
```

File: cypress/e2e/admin/admin/AdminSelecting.cy.js [hunk 12]
```diff
@@ -0,0 +1,27 @@
...
+    cy.visit("/admin/admin");
+  });
+
+  it("Select All Click Action", () => {
+    cy.get('[data-cy="select_all"]').click();
+    adminList.forEach((admin) => {
+      cy.get(`[data-cy="${admin.uid}"]`).should("have.class", "bg-green-100");
+    });
+  });
+  it("Select First Five Entries Action", () => {
```
Comment (shahdivyank) [inline cypress/e2e/admin/admin/AdminSelecting.cy.js]:
```suggestion

 it("Select First Five Entries Action", () => {
```

File: cypress/e2e/admin/admin/AdminActions.cy.js [hunk 13]
```diff
@@ -0,0 +1,52 @@
...
+        .should("exist");
+  });
+
+  it("Reject First Five Entries Action", () => {
+    for (let i = 0; i < 5; i++)
+      cy.get(`[data-cy="${adminList[i].uid}"]`)
+        .find('[data-cy="checkbox"]')
+        .click();
+    cy.get('[data-cy="toolbar"]').find('[data-cy="reject_tag"]').click();
+    for (let i = 0; i < 5; i++)
```
Comment (shahdivyank) [inline cypress/e2e/admin/admin/AdminActions.cy.js]: use forEach loop

File: cypress/e2e/admin/admin/AdminActions.cy.js [hunk 14]
```diff
@@ -0,0 +1,44 @@
+import { adminList } from "../../../../src/data/mock/Admin";
+describe("Admin Actions", () => {
+  beforeEach(() => {
+    cy.login("admin");
+    cy.visit("/");
+    cy.wait("@session");
+    cy.visit("/admin/admin");
+  });
+  it("Accept First Five Entries Action", () => {
```
Comment (shahdivyank) [inline cypress/e2e/admin/admin/AdminActions.cy.js]:
```suggestion
  it("Accept First Five Entries", () => {
```

File: cypress/e2e/admin/admin/AdminActions.cy.js [hunk 15]
```diff
@@ -0,0 +1,44 @@
...
+        .find('[data-cy="checkbox"]')
+        .click();
+    cy.get('[data-cy="toolbar"]').find('[data-cy="accept-tag"]').click();
+    for (let i = 0; i < 5; i++)
+      cy.get(`[data-cy="${adminList[i].uid}"]`)
+        .find('[data-cy="accepted-tag"]')
+        .should("exist");
+  });
+
+  it("Reject First Five Entries Action", () => {
```
Comment (shahdivyank) [inline cypress/e2e/admin/admin/AdminActions.cy.js]:
```suggestion
  it("Reject First Five Entries", () => {
```

File: cypress/e2e/admin/admin/AdminActions.cy.js [hunk 16]
```diff
@@ -0,0 +1,44 @@
...
+        .find('[data-cy="checkbox"]')
+        .click();
+    cy.get('[data-cy="toolbar"]').find('[data-cy="reject-tag"]').click();
+    for (let i = 0; i < 5; i++)
+      cy.get(`[data-cy="${adminList[i].uid}"]`)
+        .find('[data-cy="rejected-tag"]')
+        .should("exist");
+  });
+
+  it("Delete First Five Action", () => {
```
Comment (shahdivyank) [inline cypress/e2e/admin/admin/AdminActions.cy.js]:
```suggestion
  it("Delete First Five", () => {
```

File: cypress/e2e/admin/admin/AdminFilter.cy.js [hunk 17]
```diff
@@ -0,0 +1,112 @@
+import { adminList } from "../../../../src/data/mock/Admin";
+
+describe("Admin Filter", () => {
```
Comment (shahdivyank) [inline cypress/e2e/admin/admin/AdminFilter.cy.js]:
```suggestion
describe("Admin Filters", () => {
```

File: cypress/e2e/admin/admin/AdminFilter.cy.js [hunk 18]
```diff
@@ -0,0 +1,112 @@
...
+  beforeEach(() => {
+    cy.login("admin");
+    cy.visit("/");
+    cy.wait("@session");
+    cy.visit("/admin/admin");
+  });
+
+  it("Filter Default Color", () => {
```
Comment (shahdivyank) [inline cypress/e2e/admin/admin/AdminFilter.cy.js]:
```suggestion
  it("Default Filters", () => {
```

File: cypress/e2e/admin/admin/AdminFilter.cy.js [hunk 19]
```diff
@@ -0,0 +1,112 @@
...
+      .should("have.class", "text-white");
+    cy.get('[data-cy="accepted-filter"]')
+      .get("div")
+      .should("have.class", "bg-hackathon-blue-100");
+    cy.get('[data-cy="accepted-filter"]')
+      .get("div")
+      .should("have.class", "text-white");
+  });
+
+  it("Filters Click Color", () => {
```
Comment (shahdivyank) [inline cypress/e2e/admin/admin/AdminFilter.cy.js]:
```suggestion
  it("Click Filters", () => {
```

File: cypress/e2e/admin/admin/AdminFilter.cy.js [hunk 20]
```diff
@@ -0,0 +1,112 @@
...
+    cy.get('[data-cy="accepted-filter"]').click();
+    cy.get('[data-cy="accepted-filter"]')
+      .get("div")
+      .should("have.class", "bg-white");
+    cy.get('[data-cy="accepted-filter"]')
+      .get("div")
+      .should("have.class", "text-hackathon-blue-100");
+  });
+
+  it("Pending Click Action", () => {
```
Comment (shahdivyank) [inline cypress/e2e/admin/admin/AdminFilter.cy.js]:
```suggestion
  it("Click Pending", () => {
```

File: cypress/e2e/admin/admin/AdminFilter.cy.js [hunk 21]
```diff
@@ -0,0 +1,112 @@
...
+    cy.get('[data-cy="pending-filter"]').click();
+    adminList.forEach((admin) => {
+      if (admin.status === "pending")
+        cy.get(`[data-cy="${admin.uid}"]`).should("not.exist");
+      else cy.get(`[data-cy="${admin.uid}"]`).should("exist");
+    });
+  });
+
+  it("Rejected Click Action", () => {
```
Comment (shahdivyank) [inline cypress/e2e/admin/admin/AdminFilter.cy.js]:
```suggestion
  it("Click Rejected", () => {
```

File: cypress/e2e/admin/admin/AdminFilter.cy.js [hunk 22]
```diff
@@ -0,0 +1,112 @@
...
+    cy.get('[data-cy="rejected-filter"]').click();
+    adminList.forEach((admin) => {
+      if (admin.status === "reject")
+        cy.get(`[data-cy="${admin.uid}"]`).should("not.exist");
+      else cy.get(`[data-cy="${admin.uid}"]`).should("exist");
+    });
+  });
+
+  it("Accepted Click Action", () => {
```
Comment (shahdivyank) [inline cypress/e2e/admin/admin/AdminFilter.cy.js]:
```suggestion
  it("Click Accepted", () => {
```

File: cypress/e2e/admin/admin/AdminFilter.cy.js [hunk 23]
```diff
@@ -0,0 +1,112 @@
...
+    cy.get('[data-cy="accepted-filter"]').click();
+    adminList.forEach((admin) => {
+      if (admin.status === "accept")
+        cy.get(`[data-cy="${admin.uid}"]`).should("not.exist");
+      else cy.get(`[data-cy="${admin.uid}"]`).should("exist");
+    });
+  });
+
+  it("Pending Accepted Click Action", () => {
```
Comment (shahdivyank) [inline cypress/e2e/admin/admin/AdminFilter.cy.js]:
```suggestion
  it("Click Pending and Accepted", () => {
```

File: cypress/e2e/admin/admin/AdminFilter.cy.js [hunk 24]
```diff
@@ -0,0 +1,112 @@
...
+    cy.get('[data-cy="accepted-filter"]').click();
+    cy.get('[data-cy="pending-filter"]').click();
+    adminList.forEach((admin) => {
+      if (admin.status === "accept" || admin.status === "pending")
+        cy.get(`[data-cy="${admin.uid}"]`).should("not.exist");
+      else cy.get(`[data-cy="${admin.uid}"]`).should("exist");
+    });
+  });
+
+  it("Pending Rejected Click Action", () => {
```
Comment (shahdivyank) [inline cypress/e2e/admin/admin/AdminFilter.cy.js]:
```suggestion
  it("Click Pending and Rejected", () => {
```

File: cypress/e2e/admin/admin/AdminFilter.cy.js [hunk 25]
```diff
@@ -0,0 +1,112 @@
...
+    cy.get('[data-cy="rejected-filter"]').click();
+    cy.get('[data-cy="pending-filter"]').click();
+    adminList.forEach((admin) => {
+      if (admin.status === "reject" || admin.status === "pending")
+        cy.get(`[data-cy="${admin.uid}"]`).should("not.exist");
+      else cy.get(`[data-cy="${admin.uid}"]`).should("exist");
+    });
+  });
+
+  it("Accepted Rejected Click Action", () => {
```
Comment (shahdivyank) [inline cypress/e2e/admin/admin/AdminFilter.cy.js]:
```suggestion
  it("Click Accepted and Rejected", () => {
```

File: cypress/e2e/admin/admin/AdminSearching.cy.js [hunk 26]
```diff
@@ -0,0 +1,22 @@
+import { adminList } from "../../../../src/data/mock/Admin";
+
+describe("Admin Searching", () => {
```
Comment (shahdivyank) [inline cypress/e2e/admin/admin/AdminSearching.cy.js]:
```suggestion
describe("Admin Search", () => {
```

File: cypress/e2e/admin/admin/AdminSearching.cy.js [hunk 27]
```diff
@@ -0,0 +1,22 @@
...
+  beforeEach(() => {
+    cy.login("admin");
+    cy.visit("/");
+    cy.wait("@session");
+    cy.visit("/admin/admin");
+  });
+
+  it("Input Not Exist Action", () => {
```
Comment (shahdivyank) [inline cypress/e2e/admin/admin/AdminSearching.cy.js:11]:
```suggestion
  it("Empty Input", () => {
```

Comment (shahdivyank) [inline cypress/e2e/admin/admin/AdminSearching.cy.js]:
```suggestion
  it("No Search Results", () => {
```

File: cypress/e2e/admin/admin/AdminSearching.cy.js [hunk 28]
```diff
@@ -0,0 +1,22 @@
...
+    cy.get('[data-cy="toolbar"]').find("input").type("Meow");
+    cy.get('[data-cy="toolbar"]').find("form").submit();
+    cy.contains("No admin Available");
+  });
+
+  it("Search For The First Entry Action", () => {
```
Comment (shahdivyank) [inline cypress/e2e/admin/admin/AdminSearching.cy.js]:
```suggestion
  it("Search For First Entry", () => {
```

File: cypress/e2e/admin/admin/AdminSelecting.cy.js [hunk 29]
```diff
@@ -0,0 +1,29 @@
+import { adminList } from "../../../../src/data/mock/Admin";
+
+describe("Admin Selecting", () => {
```
Comment (shahdivyank) [inline cypress/e2e/admin/admin/AdminSelecting.cy.js]:
```suggestion
describe("Admin Select", () => {
```

File: cypress/e2e/admin/admin/AdminSelecting.cy.js [hunk 30]
```diff
@@ -0,0 +1,29 @@
...
+  beforeEach(() => {
+    cy.login("admin");
+    cy.visit("/");
+    cy.wait("@session");
+    cy.visit("/admin/admin");
+  });
+
+  it("Select All Click Action", () => {
```
Comment (shahdivyank) [inline cypress/e2e/admin/admin/AdminSelecting.cy.js]:
```suggestion
  it("Select All", () => {
```

File: cypress/e2e/admin/admin/AdminSelecting.cy.js [hunk 31]
```diff
@@ -0,0 +1,29 @@
...
+    cy.get('[data-cy="select-all"]').click();
+    adminList.forEach((admin) => {
+      cy.get(`[data-cy="${admin.uid}"]`).should("have.class", "bg-green-100");
+    });
+  });
+
+  it("Select First Five Entries Action", () => {
```
Comment (shahdivyank) [inline cypress/e2e/admin/admin/AdminSelecting.cy.js]:
```suggestion
  it("Select First Five Entries", () => {
```

File: cypress/downloads/downloads.html [hunk 32]
```diff
@@ -0,0 +1 @@
+Cr24
```
Comment (shahdivyank) [inline cypress/downloads/downloads.html]: what is this?

File: cypress/e2e/admin/admin/AdminActions.cy.js [hunk 33]
```diff
@@ -0,0 +1,44 @@
+import { adminList } from "../../../../src/data/mock/Admin";
+describe("Admin Actions", () => {
+  beforeEach(() => {
+    cy.login("admin");
+    cy.visit("/");
+    cy.wait("@session");
+    cy.visit("/admin/admin");
+  });
+  it("Accept First Five Entries", () => {
```
Comment (shahdivyank) [inline cypress/e2e/admin/admin/AdminActions.cy.js]:
```suggestion
  it("Accept First 5 Entries", () => {
```

File: cypress/e2e/admin/admin/AdminActions.cy.js [hunk 34]
```diff
@@ -0,0 +1,44 @@
...
+        .find('[data-cy="checkbox"]')
+        .click();
+    cy.get('[data-cy="toolbar"]').find('[data-cy="accept-tag"]').click();
+    for (let i = 0; i < 5; i++)
+      cy.get(`[data-cy="${adminList[i].uid}"]`)
+        .find('[data-cy="accepted-tag"]')
+        .should("exist");
+  });
+
+  it("Reject First Five Entries", () => {
```
Comment (shahdivyank) [inline cypress/e2e/admin/admin/AdminActions.cy.js]:
```suggestion
  it("Reject First 5 Entries", () => {
```

File: cypress/e2e/admin/admin/AdminActions.cy.js [hunk 35]
```diff
@@ -0,0 +1,44 @@
...
+        .find('[data-cy="checkbox"]')
+        .click();
+    cy.get('[data-cy="toolbar"]').find('[data-cy="reject-tag"]').click();
+    for (let i = 0; i < 5; i++)
+      cy.get(`[data-cy="${adminList[i].uid}"]`)
+        .find('[data-cy="rejected-tag"]')
+        .should("exist");
+  });
+
+  it("Delete First Five", () => {
```
Comment (shahdivyank) [inline cypress/e2e/admin/admin/AdminActions.cy.js]:
```suggestion
  it("Delete First 5 Entries", () => {
```

File: cypress/e2e/admin/admin/AdminSearching.cy.js [hunk 36]
```diff
@@ -0,0 +1,22 @@
...
+    cy.visit("/admin/admin");
+  });
+
+  it("No Search Results", () => {
+    cy.get('[data-cy="toolbar"]').find("input").type("Meow");
+    cy.get('[data-cy="toolbar"]').find("form").submit();
+    cy.contains("No admin Available");
+  });
+
+  it("Search For First Entry", () => {
```
Comment (shahdivyank) [inline cypress/e2e/admin/admin/AdminSearching.cy.js]:
```suggestion
  it("Search For 1st Entry", () => {
```

File: cypress/e2e/admin/admin/AdminSelecting.cy.js [hunk 37]
```diff
@@ -0,0 +1,29 @@
...
+  });
+
+  it("Select All", () => {
+    cy.get('[data-cy="select-all"]').click();
+    adminList.forEach((admin) => {
+      cy.get(`[data-cy="${admin.uid}"]`).should("have.class", "bg-green-100");
+    });
+  });
+
+  it("Select First Five Entries", () => {
```
Comment (shahdivyank) [inline cypress/e2e/admin/admin/AdminSelecting.cy.js]:
```suggestion
  it("Select First 5 Entries", () => {
```

File: cypress/e2e/admin/admin/AdminActions.cy.js [hunk 38]
```diff
@@ -0,0 +1,44 @@
+import { adminList } from "../../../../src/data/mock/Admin";
+describe("Admin Actions", () => {
+  beforeEach(() => {
+    cy.login("admin");
+    cy.visit("/");
+    cy.wait("@session");
+    cy.visit("/admin/admin");
+  });
```
Comment (shahdivyank) [inline cypress/e2e/admin/admin/AdminActions.cy.js]:
```suggestion
  });
  
```

File: cypress/e2e/admin/admin/Actions.cy.js [hunk 39]
```diff
@@ -1,4 +1,5 @@
 import { adminList } from "../../../../src/data/mock/Admin";
+const first5Admin = adminList.slice(0, 5);
```
Comment (shahdivyank) [inline cypress/e2e/admin/admin/Actions.cy.js]: just call it five

File: cypress/e2e/admin/admin/Selecting.cy.js [hunk 40]
```diff
@@ -16,10 +16,14 @@ describe("Admin Select", () => {
   });
 
   it("Select First 5 Entries", () => {
-    for (let i = 0; i < 5; i++)
-      cy.get(`[data-cy="${adminList[i].uid}"]`)
-        .find('[data-cy="checkbox"]')
-        .click();
+    adminList
+      .slice(0, 5)
```
Comment (shahdivyank) [inline cypress/e2e/admin/admin/Selecting.cy.js]: do this slice at the top

File: cypress/e2e/admin/admin/Actions.cy.js (hunk 39 above)
Comment (shahdivyank) [reply cypress/e2e/admin/admin/Actions.cy.js]: or fiveAdmins if you want

File: cypress/e2e/admin/admin/Actions.cy.js [hunk 41]
```diff
@@ -0,0 +1,49 @@
+import { adminList } from "../../../../src/data/mock/Admin";
+const five = adminList.slice(0, 5);
+describe("Admin Actions", () => {
```
Comment (shahdivyank) [inline cypress/e2e/admin/admin/Actions.cy.js:5]:
```suggestion

describe("Admin Actions", () => {
```

File: cypress/e2e/admin/admin/Actions.cy.js [hunk 42]
```diff
@@ -0,0 +1,49 @@
+import { adminList } from "../../../../src/data/mock/Admin";
```
Comment (shahdivyank) [inline cypress/e2e/admin/admin/Actions.cy.js:1]:
```suggestion

import { adminList } from "../../../../src/data/mock/Admin";
```

File: cypress/e2e/admin/admin/Actions.cy.js [hunk 43]
```diff
@@ -0,0 +1,49 @@
...
+const five = adminList.slice(0, 5);
```
Comment (shahdivyank) [inline cypress/e2e/admin/admin/Actions.cy.js:3]:
```suggestion

const five = adminList.slice(0, 5);
```

File: cypress/e2e/admin/admin/Actions.cy.js [hunk 44]
```diff
@@ -0,0 +1,49 @@
...
+    five.forEach((admin) =>
+      cy
+        .get(`[data-cy="${admin.uid}"]`)
+        .find('[data-cy="rejected-tag"]')
+        .should("exist")
+    );
+  });
+
+  it("Delete First 5 Entries", () => {
+    const deleteList = five.slice(0, 5);
```
Comment (shahdivyank) [inline cypress/e2e/admin/admin/Actions.cy.js]: this is pointless?!

File: cypress/e2e/admin/admin/Select.cy.js [hunk 45]
```diff
@@ -0,0 +1,30 @@
...
+  it("Select All", () => {
+    cy.get('[data-cy="select-all"]').click();
+    adminList.forEach((admin) => {
+      cy.get(`[data-cy="${admin.uid}"]`).should("have.class", "bg-green-100");
+    });
+  });
+
+  it("Select First 5 Entries", () => {
+    adminList
+      .slice(0, 5)
```
Comment (shahdivyank) [inline cypress/e2e/admin/admin/Select.cy.js]: do this at the top

File: cypress/e2e/admin/admin/Select.cy.js [hunk 46]
```diff
@@ -0,0 +1,29 @@
+import { adminList } from "../../../../src/data/mock/Admin";
```
Comment (shahdivyank) [inline cypress/e2e/admin/admin/Select.cy.js:1]:
```suggestion
import { adminList } from "../../../../src/data/mock/Admin";

```

**Review by shahdivyank — APPROVED** [review]

**Review by rfairooz — APPROVED** [review]

**Pull request merged**